boards/nxp/frdm_mcxn947/doc/index.rst

```diff
@@ -259,6 +310,12 @@ should see the following message in the terminal:
    *** Booting Zephyr OS build v3.6.0-479-g91faa20c6741 ***
    Hello World! frdm_mcxn947/mcxn947/cpu0
 
+Debugging a dual-core image
+---------------------------
+
+For dual core builds, the secondary core should be placed into a loop,
+then a debugger can be attached (see `AN13264`_, section 4.2.3 for more information)
```

Comment:
Sorry, should have been more clear here. AN13264 is specifically about the RT1170- do we have an appnote like this for the MCXN? If not, we can probably link to this appnote and say "using a technique similar to that described for the RT1170 in AN13264"